datasets/lt/pep_declarations/crawler.py

```diff
@@ -0,0 +1,204 @@
+            )
+
+            categorisation = categorise(
+                context,
+                position,
+                is_pep=is_pep,
+            )
+
+            occupancy = h.make_occupancy(
+                context,
+                person,
+                position,
+                no_end_implies_current=False,
+                start_date=affiliation_start_date,
+                categorisation=categorisation,
+                propagate_country=True,
+            )
+            context.audit_data(role)
+            if occupancy:
+                entities.append((position, occupancy))
+        context.audit_data(data)
+    return entities
+
+
+def crawl(context: Context) -> None:
+    """exhaustively scans PINREG portal and emits all deklaracijos"""
+
+    pinreg = PinregSession(context)
+    for deklaracija_id in DEKLARACIJA_ID_RANGE:
+        sleep(0.3)
+        if not (record := pinreg.get_deklaracija_by_id(deklaracija_id)):
+            continue
+
+        assert record.pop("id") == deklaracija_id
+
+        # metadata
+        submission_date = record.pop("pateikimoData")
+        modified_at = record.pop("viesumoNutraukimoData")
+        reasons_for_submission = record.pop("teikimoPriezastysPavadinimai")
+        submission_status = record.pop("viesumoStatusas")
+        nondisclosure_reason = record.pop("neviesinimoPriezastis")
+        declaration_type = record.pop("deklaracijosTipas")
+
+        # declarant data
+        declarant = make_person(context, record.pop("teikejas"))
+        declarant_affiliations: list[dict] = record.pop("darbovietes")
+        declarant_offices = parse_affiliations(
+            context, declarant, declarant_affiliations
+        )
+        for position, occupancy in declarant_offices:
+            context.emit(position, target=False)
+            context.emit(occupancy, target=False)
+        if not declarant_offices:
+            context.log.info(
+                f"deklaracija {deklaracija_id} has no relevant occupancies."
+            )
+            continue
+        context.log.info(f"deklaracija {deklaracija_id} processing.")
+
+        """If the spouse is included in the declaration, and the spouse's occupancies
+        meet OpenSanctions criteria, emit the spouse, the marriage, and the occupancies/positions."""
+        spouse_affiliations: list[dict] = record.pop("sutuoktinioDarbovietes", [])
+        if spouse_data := record.pop("sutuoktinis", None):
+            spouse = make_person(context, spouse_data) if spouse_data else None
+            marriage = make_spouse(context, declarant, spouse)
+            spouse_offices = parse_affiliations(context, spouse, spouse_affiliations)
+            if spouse_offices:
+                for position, occupancy in spouse_offices:
+                    context.emit(position, target=False)
+                    context.emit(occupancy, target=False)
+            context.emit(spouse, target=False)
+            context.emit(marriage)
```

Comment:
we don't want positions and occupancies for the spouse - we only want the spouse and family relation.

If the spouse is a declarant, they'll have their own declaration so we won't miss them.